apps/io-func-wallet-solution-backend/src/infra/attestation-service/index.ts

```diff
@@ -123,7 +124,8 @@ export class MobileAttestationService implements AttestationService {
               RA.head,
               E.fromOption(() => new Error("No assertion validation passed"))
             )
-          )
+          ),
+          RTE.fromTaskEither
```

Comment:
Why did it become an `RTE`?

Reply:
because the first element of the `pipe` is now an `RTE` (since `walletInstance: getWalletInstance(hardwareKeyTag, userId)` has now the database dependency)

Reply:
Ok, what do you think about moving DB access to the calling function? In this way we leave the attestation service independent of the database. And you might think about passing hardwareKey and signCount as parameters to the function. Let me know your opinion on it.